src/commands/webViewCommand.ts

```diff
@@ -0,0 +1,156 @@
+        const detailsDetachedView = new AstDetailsDetached(
+          this.context.extensionUri,
+          result,
+          this.context,
+          false,
+          type
+        );
+        // Need to check if the detailsPanel is positioned in the rigth place
+        if (
+          this.detailsPanel?.viewColumn === 1 ||
+          !this.detailsPanel?.viewColumn
+        ) {
+          this.detailsPanel?.dispose();
+          this.detailsPanel = undefined;
+          await vscode.commands.executeCommand(
+            "workbench.action.splitEditorRight"
+          );
+          // Only keep the result details in the split
+          await vscode.commands.executeCommand(
+            "workbench.action.closeEditorsInGroup"
+          );
+        }
+        this.detailsPanel?.dispose();
+        this.detailsPanel = vscode.window.createWebviewPanel(
+          "newDetails", // Identifies the type of the webview, internal id
+          "(" + result.severity + ") " + result.label.replaceAll("_", " "), // Title of the detailsPanel displayed to the user
+          vscode.ViewColumn.Two, // Show the results in a separated column
+          {
+            enableScripts: true,
+            localResourceRoots: [
+              vscode.Uri.file(path.join(this.context.extensionPath, "media")),
+            ],
+          }
+        );
+        // Only allow one detail to be open
+        this.detailsPanel.onDidDispose(
+          () => {
+            this.detailsPanel = undefined;
+          },
+          null,
+          this.context.subscriptions
+        );
+        // detailsPanel set options
+        this.detailsPanel.webview.options = {
+          enableScripts: true,
+          localResourceRoots: [
+            vscode.Uri.file(path.join(this.context.extensionPath, "media/")),
+          ],
+        };
+        // detailsPanel set html content
+        this.detailsPanel.webview.html =
+          await detailsDetachedView.getDetailsWebviewContent(
+            this.detailsPanel.webview
+          );
+
+        // Start to load the changes tab, gets called everytime a new sast details webview is opened
+        if (result.type === "sast") {
+          await getLearnMore(
+            this.logs,
+            this.context,
+            result,
+            this.detailsPanel
+          );
+        }
+        if (result.type === "sast" || result.type === "kics") {
+          await getChanges(this.logs, this.context, result, this.detailsPanel);
+        }
+        // Start to load the bfl, gets called everytime a new details webview is opened in a SAST result
+        //result.sastNodes.length>0 && getResultsBfl(logs,context,result,detailsPanel);
+        // Comunication between webview and extension
+        this.detailsPanel.webview.onDidReceiveMessage(async (data) => {
+          switch (data.command) {
+            // Catch open file message to open and view the result entry
+            case "showFile":
```

Comment:
@tiagobcx  The "showFile" value should be constant and use in the webview if possible.